### PR TITLE
Added recipe for fortranformat python package

### PR DESCRIPTION
**Pull request by smithsp**

### CODE REVIEW

File: recipes/fortranformat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - fortranformat
+
+about:
+  home: http://bitbucket.org/brendanarnold/py-fortranformat
+  license: MIT
+  license_family: MIT
+  license_file: ''
```
Comment (scopatz) [inline recipes/fortranformat/meta.yaml]: License file shouldn't be an empty string

File: recipes/fortranformat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    \nTo read Fortran records,:\n\n  >>> import fortranformat as ff\n  >>> header_line = ff.FortranRecordReader('(A15, A15, A15)')\n  >>> header_line.read('              x              y              z')\n\
+    \    ['              x', '              y', '              z']\n  >>> line = FortranRecordReader('(3F15.3)')\n  >>> line.read('          1.000          0.000          0.500')\n    [1.0, 0.0, 0.5]\n\
+    \  >>> line.read('          1.100          0.100          0.600')\n    [1.1, 0.1, 0.6]\n\nTo write Fortran records,:\n\n  >>> import fortranformat as ff\n  >>> header_line = ff.FortranRecordWriter('(A15,\
+    \ A15, A15)')\n  >>> header_line.write(['x', 'y', 'z'])\n    '              x              y              z'\n  >>> line = FortranRecordWriter('(3F15.3)')\n  >>> line.write([1.0, 0.0, 0.5])\n    ' \
+    \         1.000          0.000          0.500'\n  >>> line.write([1.1, 0.1, 0.6])\n    '          1.100          0.100          0.600'"
+  doc_url: ''
+  dev_url: ''
```
Comment (scopatz) [inline recipes/fortranformat/meta.yaml]: Can you clean up the description and remove the empty strings please? Thanks!

**Comment by smithsp** [comment]
@scopatz Thanks for the feedback.

**Comment by smithsp** [comment]
@synapticarbors Thank you for the review.

**Comment by smithsp** [comment]
Thanks @synapticarbors

**Pull request merged**